llvm/include/llvm/TargetParser/Triple.h

```diff
@@ -769,7 +769,7 @@ class Triple {
   bool isOSGlibc() const {
     return (getOS() == Triple::Linux || getOS() == Triple::KFreeBSD ||
             getOS() == Triple::Hurd) &&
-           !isAndroid();
+           !isAndroid() && !isMusl();
```

Comment:
Are the other callers ok with this? Looks like it affects ABI stuff on some targets?